### ISSUE TITLE
Can't select anything inside window

### ISSUE THREAD

**Issue by spauldingdrive** [acjim/angular-html-window#2]
I think it's z-index bug. Here is an example: http://jsfiddle.net/emvc79sa/1/

**Comment by rlamana** [comment]
I don't think it is a z-index bug. I'm pretty sure this is a Ventus issue due to all these 'user-select: none' in the CSS: https://github.com/rlamana/Ventus/blob/master/src/ventus/less/window.less#L43

The original idea of the library was to behave like a normal window manager outside the web, so window content by default is non-selectable.

**Comment by jackkum** [comment]
This could help for someone. 
I deleted this line https://github.com/acjim/angular-html-window/blob/master/src/ngHtmlWindow.js#L211
